src/Aspire.Hosting.Azure/Provisioning/Provisioners/AzureProvisioner.cs

```diff
@@ -378,25 +372,25 @@ await foreach (var tenant in armClient.GetTenants().GetAllAsync(cancellationToke
         AzureLocation location = new(_options.Location);
         try
         {
-            var response = await resourceGroups.GetAsync(resourceGroupName, cancellationToken).ConfigureAwait(false);
+            var response = await resourceGroups.GetAsync(_options.ResourceGroup, cancellationToken).ConfigureAwait(false);
             resourceGroup = response.Value;
 
             logger.LogInformation("Using existing resource group {rgName}.", resourceGroup.Data.Name);
         }
         catch (Exception)
         {
-            if (!createIfAbsent)
+            if (!_options.AllowResourceGroupCreation ?? false)
```

Comment:
Does this mean that I need to also set `AllowResourceGroupCreation` in my config in order to get it created? This wasn't necessary before.